### ISSUE TITLE
How do I add multiple Youless meters to Home Assistant?

### ISSUE THREAD

**Issue by huub66** [reharmsen/hass-youless-component#9]
With following configuration.yaml sensor configuration, I am only seeing 1 youless sensor created, the Youless meter for the heatpump isn't visible. How can I add multiple Youless meters to my Home Assistant instance? I have 3 in total which I'd like to add.

  - platform: youless
    name: Youless
    host: 192.168.2.29
    monitored_variables:
      - pwr
      - net
      - p1
      - p2
      - n1
      - n2
      - gas
  - platform: youless
    name: Youless Heatpump
    host: 192.168.2.28
    monitored_variables:
      - pwr
      - cs0
      - ps0

Following error in log; "Entity id already exists - ignoring: sensor.youless_current_power_usage"

**Comment by gjong** [comment]
If you have a little patience then Youless will be natively supported by Home Assistant. Including multiple devices. See the pull request https://github.com/home-assistant/core/pull/41942 that will add support.

**Comment by huub66** [comment]
> 
> 
> If you have a little patience then Youless will be natively supported by Home Assistant. Including multiple devices. See the pull request [home-assistant/core#41942](https://github.com/home-assistant/core/pull/41942) that will add support.

looking forward to that! what kind of timeline are we looking at before native support will be ready?

**Comment by tiger42** [comment]
@huub66: As long as the native Youless support is not integrated into HA, I have forked this project and created an interim solution: https://github.com/tiger42/hass-youless-component

**Comment by gjong** [comment]
@huub66 if you want to already give the new integration a shot the you can get the latest version at https://bitbucket.org/jongsoftdev/youless/src/master/.

I keep this up to date with the version that will be integrated into the official home assistant. It will allow you to try it out. And help me to get the final native version better, especially if you report any issues.

**Comment by reharmsen** [comment]
Sorry for not seeing this more early.
I am not that much into python or home assistant component development, so I am not able to help with making this work for multiple instances.

I just added some (small) stuff to get HACS integration.. I also just fixed a small thing to be able to keep this working after 2021.6

Would be nice to have native support though... when that integrates and gets released I probably will close up this repo.

So for now waiting for native support
